### PR TITLE
Updating copyright year

### PR DESCRIPTION
**Pull request by birbalsain**
#### Bug/feature Description
* coptright year of  pbspro has changed

#### Affected Platform(s)
* all

#### Cause / Analysis / Design
* copyright year of pbspro has changed

#### Solution Description
* copyright year of pbspro is changed to 2019

#### Testing logs/output
* [copyright_year.txt](https://github.com/PBSPro/pbspro/files/2875596/copyright_year.txt)

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: test/fw/doc/conf.py [hunk 1]
```diff
@@ -246,7 +246,7 @@
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [('index', 'PbsTestLab.tex', u'PbsTestLab Documentation',
-                    u'Copyright (C) 1994-2018 Altair Engineering, Inc',
```
Comment (vstumpf) [inline test/fw/doc/conf.py]: Travis is failing because the end quote was removed.

Comment (mike0042) [reply test/fw/doc/conf.py]: There are two additional instances of this at lines 278 and 290 in this file.

**Comment by prakashcv13** [comment]
AppVeyor debug is failing - I restarted the build, that too failed.

**Comment by sakshamgarg** [comment]
LICENSE file is not updated. It has lower case 'c' inside the brackets of "Copyright(c)".

**Comment by birbalsain** [comment]
Thanks, @sakshamgarg for your approval,
@PBSPro/pbspro-maintainers please take this PR further.

**Comment by birbalsain** [comment]
@prakashcv13  please take this PR further.

**Comment by prakashcv13** [comment]
@birbalsain - thank you for making the changes. Please squash and rebase for merge.

**Comment by prakashcv13** [comment]
@birbalsain - please sign your commit.

**Comment by prakashcv13** [comment]
merged #994

**Pull request merged**